### PR TITLE
Render table comments

### PR DESCRIPTION
**Pull request by dhirschfeld**

### CODE REVIEW

**Comment by dhirschfeld** [comment]
The new(-ish) column comments feature is fantastic! ❤️ 

...so much so that I'd really like for table comments to also be reflected. Unfortunately when https://github.com/agronholm/sqlacodegen/commit/fa3c4a9cee8588f1d34fc65467122bc1b5b1006e was merged the table comment feature from #92 wasn't also ported over.

This PR just includes the missing table comment support from #92.

**Comment by dhirschfeld** [comment]
Looks like the only failure on py38 is unrelated:
```
Setting environment variables from .travis.yml
$ export TOXENV=py38
0.00s$ source ~/virtualenv/python3.8-dev/bin/activate
$ python --version
Python 3.8.1+
$ pip --version
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.8-dev/bin/pip", line 6, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
0.02s$ pip install tox
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.8-dev/bin/pip", line 6, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
The command "pip install tox" failed and exited with 1 during .
```

**Comment by dhirschfeld** [comment]
So, that didn't fix it:
```
Setting environment variables from .travis.yml
$ export TOXENV=py38
0.00s$ source ~/virtualenv/python3.8-dev/bin/activate
$ python --version
Python 3.8.1+
$ pip --version
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.8-dev/bin/pip", line 6, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
0.02s$ python -m pip install tox
/home/travis/virtualenv/python3.8-dev/bin/python: No module named pip
```

Was worth a try I guess.

**Comment by dhirschfeld** [comment]
Looks like others are running into problems with travis too:
https://gitter.im/python-trio/general?at=5e424170f3718e705df566f7

**Comment by dhirschfeld** [comment]
Test at least seems to work locally:
image

**Comment by dhirschfeld** [comment]
Hopefully it's fixed now:
https://github.com/pypa/virtualenv/issues/1545#issuecomment-584779522

**Comment by dhirschfeld** [comment]
Ping @agronholm! CI (and coverage) now pass 🎉 

It would be great to get this in so as to have self-documenting models.

File: tests/test_codegen.py [hunk 1]
```diff
@@ -1285,6 +1286,41 @@ class Simple(Base):
...
+def test_table_comment(metadata):
+    Table(
+        'simple', metadata,
+        Column('id', INTEGER, primary_key=True),
+        comment="this is a 'comment'"
+    )
+
+    codegen = CodeGenerator(metadata)
+    code = codegen.render_table(codegen.models[0])
+    expected = dedent("""
```
Comment (agronholm) [inline tests/test_codegen.py]: Why use dedent here specifically and not elsewhere?

Comment (dhirschfeld) [reply tests/test_codegen.py]: No particularly good reason - I can revert that.

**Comment by agronholm** [comment]
I'll merge this if you fix the remaining errors.

**Comment by dhirschfeld** [comment]
All green!

**Comment by agronholm** [comment]
Thanks!

**Comment by dhirschfeld** [comment]
Thank you @agronholm!

**Pull request merged**